gson/build.gradle

```diff
@@ -4,8 +4,8 @@ apply plugin: 'maven'
 group = 'com.google.code.gson'
 version = '2.8.6-SNAPSHOT'
 
-sourceCompatibility = 1.6
-targetCompatibility = 1.6
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
```

Comment:
Probably does not matter because the Gradle build is outdated anyway, see #1896.